components/Frontpage/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+import Link from 'next/link';
+
+const AnnouncementBanner = () => {
+  return (
+    <AnnouncementContainer>
+      <InfoContainer>
+        <StyledRow>
+          <Svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+            fill="white"
+            width="150px"
+            height="90px"
+          >
+            <path d="M0 0h24v24H0V0z" fill="none" />
+            <path d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z" />
+          </Svg>
+          <Info>
+            <StyledHeader>
+              itDAGENE 2020 er dessverre utsatt grunnet Covid-19
+            </StyledHeader>
+          </Info>
+        </StyledRow>
+        <Link href="/info/covid-19">
```

Comment:
```suggestion
        <Link href="/info/[side]" as="info/covid-19">
```
To have dynamic routing :smile:

Reply:
Good idea, I'll check it out!